nixos/modules/system/boot/stage-1-init.sh

```diff
@@ -228,6 +240,7 @@ mkdir -p /lib
 ln -s @modulesClosure@/lib/modules /lib/modules
 ln -s @modulesClosure@/lib/firmware /lib/firmware
 echo @extraUtils@/bin/modprobe > /proc/sys/kernel/modprobe
+# shellcheck disable=SC2043
```

Comment:
This disables the rule for [the rest of the file, not just the next line](https://github.com/koalaman/shellcheck/issues/960). You might want to re-enable it after the loop or pull the loop into a function to disable only this line.